Azure/communication-ui-library-android
# [Feature] Support RTL

## Purpose

* Support RTL if the Locale is Arabic 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code


1. Go to Android **Settings** and open Developers **Options Settings**
2. Under the **Drawing** settings, make sure to enable "Force RTL layout direction"** option
3. Back to App and see the RTL layout 


## What to Check
Verify that the following are valid

1. Direction Image Mirrored 
2. Layout Mirrored

## Other Information

We temporarily use dev option setting to test the feature, after Localization is done, we could use other way to test. 
|LTR - Calling Screen|RTL - Calling Screen|MS Teams RTL|
|---|---|---|
|![MicrosoftTeams-image (7)](https://user-images.githubusercontent.com/93549644/153268160-ec700f6f-1980-43a9-b15c-2f1f065d6c01.png)|![MicrosoftTeams-image (7)](https://user-images.githubusercontent.com/93549644/153965254-8640c2be-30ce-4121-9a16-be9473741724.png)|![MicrosoftTeams-image (6)](https://user-images.githubusercontent.com/93549644/153965384-66b32fad-5f16-4019-a167-69cf37323d28.png)|
